samples/GalleryApp/README.md

```diff
@@ -0,0 +1,118 @@
+# GalleryApp
+
+## Summary
+The Gallery App contains a set of vision skills released on WindowsVisionSkillsPreview. It demonstrates the capabilities of the new vision skills without the need to enlist in the GitHub repo.
+
+## Getting started
+Please read the [Windows Vision Skills (Preview)](https://docs.microsoft.com/en-us/windows/ai/windows-vision-skills/) page for more detailed information about Windows Vision Skills.
+
+## Prerequisites
+- Download or modify [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+  - Under Workloads section: 
+    - Universal Windows platform development
+    - Desktop development with C++
+    - .Net desktop development
+    - .Net Core cross-platform development
+  - Select the following SDK versions: 
+    - 17134
+    - 17763
+    - 18362
```

Comment:
>18362 [](start = 6, length = 5)

this one is optional

Reply:
for compiling your app that is (we need 18362 to compile .Net Core 3.0 and C++ WinRTt Desktop apps

---
In reply to: [308046391](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/49#discussion_r308046391) [](ancestors = 308046391)